frontend/src/components/DocumentViewer/DocumentViewBoxNav.tsx

```diff
@@ -1,29 +1,28 @@
 import ThemedButton from '@src/components/ThemedButtons/ThemedButton';
 
-import './DocumentViewBoxHeader.css';
+import './DocumentViewBoxNav.css';
 
-interface DocumentViewBoxHeaderProps {
+interface DocumentViewBoxNavProps {
 	documentIndex: number;
 	documentName: string;
 	numberOfDocuments: number;
 	onPrevious: () => void;
 	onNext: () => void;
 }
 
-function DocumentViewBoxHeader({
+function DocumentViewBoxNav({
 	documentIndex,
 	documentName,
 	numberOfDocuments,
 	onPrevious,
 	onNext,
-}: DocumentViewBoxHeaderProps) {
+}: DocumentViewBoxNavProps) {
 	const documentNumber = `${documentIndex + 1} out of ${numberOfDocuments}`;
 
 	return (
-		<div className="view-documents-header">
-			<h2>View documents</h2>
+		<div className="view-documents-nav">
 			<div className="nav">
-				<div className="button-container">
+				<nav className="button-container">
```

Comment:
Does this need an aria-label? Maybe not, as it seems to be the only `<nav>` we have in our app, and it's likely self-explanatory.

Reply:
The first rule of aria: Don't use aria if you don't have to. 
Based on the context (the h1 of the container, the nav element and the buttons), it's clear what the use is, so no aria required.

Reply:
yup, just checked with NVDA on Firefox, it announces "navigation landmark" nicely 😎